src/Symfony/Component/Yaml/Parser.php

```diff
@@ -437,15 +437,15 @@ private function parseValue($value, $exceptionOnInvalidType, $objectSupport)
     }
 
     /**
-     * Parses a folded scalar.
+     * Parses a block scalar.
      *
-     * @param string $separator   The separator that was used to begin this folded scalar (| or >)
-     * @param string $indicator   The indicator that was used to begin this folded scalar (+ or -)
-     * @param int    $indentation The indentation that was used to begin this folded scalar
+     * @param string $separator   The separator that was used to begin this block scalar (| or >)
+     * @param string $chomping    The chomping indicator that was used to begin this block scalar (+ or -)
+     * @param int    $indentation The indentation indicator that was used to begin this block scalar
```

Comment:
I also think that `$separator` should be renamed `$style` (i.e. style indicator)